library/block/block_content_registry/src/main/java/org/quiltmc/qsl/block/content/registry/api/BlockContentRegistries.java

```diff
@@ -0,0 +1,132 @@
+			.builder(Registry.BLOCK,
+					new Identifier(NAMESPACE, "flattenable_block"),
+					BlockState.class,
+					BlockState.CODEC)
+			.build();
+
+	/**
+	 * A {@link RegistryEntryAttachment} for oxidizable blocks.
+	 * <p>
+	 * Values can be set via code and through a datapack with the file {@code data/quilt_block_content_registry/attachments/minecraft/block/oxdizable_block.json}
+	 */
+	public static final RegistryEntryAttachment<Block, ReversibleBlockEntry> OXIDIZABLE_BLOCK = RegistryEntryAttachment
+			.builder(Registry.BLOCK,
+					new Identifier(NAMESPACE, "oxdizable_block"),
+					ReversibleBlockEntry.class,
+					ReversibleBlockEntry.CODEC)
+			.build();
+
+	/**
+	 * A {@link RegistryEntryAttachment} for waxable blocks.
+	 * <p>
+	 * Values can be set via code and through a datapack with the file {@code data/quilt_block_content_registry/attachments/minecraft/block/waxable_block.json}
+	 */
+	public static final RegistryEntryAttachment<Block, ReversibleBlockEntry> WAXABLE_BLOCK = RegistryEntryAttachment
+			.builder(Registry.BLOCK,
+					new Identifier(NAMESPACE, "waxable_block"),
+					ReversibleBlockEntry.class,
+					ReversibleBlockEntry.CODEC)
+			.build();
+
+	/**
+	 * A {@link RegistryEntryAttachment} for strippable blocks.
+	 * <p>
+	 * Values can be set via code and through a datapack with the file {@code data/quilt_block_content_registry/attachments/minecraft/block/strippable_block.json}
+	 */
+	public static final RegistryEntryAttachment<Block, Block> STRIPPABLE_BLOCK = RegistryEntryAttachment
+			.builder(Registry.BLOCK,
+					new Identifier(NAMESPACE, "strippable_block"),
+					Block.class,
+					Registry.BLOCK.getCodec().flatXmap(block -> {
+						if (!block.getDefaultState().contains(Properties.AXIS)) {
+							return DataResult.error("block does not contain AXIS property");
+						}
+						return DataResult.success(block);
+					}, block -> {
+						if (!block.getDefaultState().contains(Properties.AXIS)) {
+							return DataResult.error("block does not contain AXIS property");
+						}
+						return DataResult.success(block);
+					}))
+			.build();
+
+	/**
+	 * A {@link RegistryEntryAttachment} for flammable blocks.
+	 * <p>
+	 * Values can be set via code and through a datapack with the file {@code data/quilt_block_content_registry/attachments/minecraft/block/flammable_block.json}
+	 */
+	public static final RegistryEntryAttachment<Block, FlammableBlockEntry> FLAMMABLE_BLOCK = RegistryEntryAttachment
+			.builder(Registry.BLOCK,
+					new Identifier(NAMESPACE, "flammable_block"),
+					FlammableBlockEntry.class,
+					FlammableBlockEntry.CODEC)
+			.build();
+
+	/**
+	 * A {@link RegistryEntryAttachment} for sculk frequencies.
+	 * <p>
+	 * Values can be set via code and through a datapack with the file {@code data/quilt_block_content_registry/attachments/minecraft/game_event/sculk_frequency.json}
+	 */
+	public static final RegistryEntryAttachment<GameEvent, Integer> SCULK_FREQUENCY = RegistryEntryAttachment
```

Comment:
There should be a testcase for sculk frequencies on the testmod

Reply:
I dont have a good way to test this as it would require adding a new game event.